etc/config.pl

```diff
@@ -390,6 +390,7 @@
             '_key' => 'paid_user', # Some things expect that key name
             '_visible_name' => 'Paid Account',
             '_account_type' => 'paid',
+            '_refund_points' => 30,
```

Comment:
Noting that these are in the prod configs (wanted that to be clear because I double-checked that that was the case / these didn't need to be added!)